src/main/resources/site/parts/frontPageBanner/frontPageBanner.html

```diff
@@ -1,7 +1,7 @@
 <section class="xp-part part-frontpage-banner">
     <div class="container">
         <div class="row">
-            <div class="col-12 col-lg-6 mx-auto mb-4">
+            <div class="col-12 col-lg-6 mx-auto mb-md-2 mt-2 text-lg-left text-center">
                 <div class="d-inline-block banner-text-wrapper">
```

Comment:
Margin top is a bit off on mobile for me still. Might have something to do with the banner-text-wrapper class.

Reply:
Changed it to use flex instead.